overlay_new.py

```diff
@@ -33,6 +34,8 @@ def data_field_coord(x, y):
 
         # Create all overlay components
         self.components = [
+            Transparent((0, 560), (330, 740)),
+            Transparent((975, 560), (1280, 740)),
```

Comment:
Rather than hardcoding these coordinates, use `self.width`, `self.height` (width and height of the display) and the coordinates for the `DataField`s so that the boxes resize dynamically if we ever use a different sized display, change the number of fields, etc.